src/main/java/mods/eln/Eln.java

```diff
@@ -4799,6 +4821,15 @@ void registerPortableItem(int id) {
 		}
 	}
 
+	private void registerFuelBurnerItem(int id) {
+		sharedItemStackOne.addElement(0 + (id << 6),
+			new FuelBurnerDescriptor(I18N.TR_NAME(Type.NONE, "Small Fuel Burner"), 5000 * fuelHeatFurnacePowerFactor, 2, 1.6f));
```

Comment:
"5000" is a magic number. How did you come up with it? Can you put that logic in the code?